.gitignore

```diff
@@ -1,3 +1,7 @@
+.env
+
 coverage/
 dist/
 node_modules/
+test-results/
```

Comment:
do we need this?

Reply:
i've added to not push the test-results to the branches, just like is usually done for coverage reports
is it better to have them?